docs/releasing/publishing.md

```diff
@@ -78,17 +78,24 @@ Developers should pair on releases. When remote working, it can be useful to be
 
 2. Sign in to npm (`npm login`), using the npm/govuk-patterns-and-tools team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/main/npm/govuk-patterns-and-tools).
 
-3. Run `npm run publish-release`, which will prompt you to either continue or cancel. Enter `y` to continue.
+3. Run `npm run publish-release`, which will prompt you to check whether the NPM tag looks as expected.
 
-4. View the created tag in the [Github interface](https://github.com/alphagov/govuk-frontend/releases) as follows:
+  If you're following these instructions, you're probably doing a sequential release and therefore
+  the tag should be "latest".
+
+  Enter 'y' to continue. If you think the tag should be different, enter `N` to have the option to set your own NPM tag.
+
+4. You will now be prompted to continue or cancel the release. Check the details and enter `y` to continue.
```

Comment:
What if they check the details and see something they need to sort before continuing?

Reply:
As above, have added a line about pressing `N` to cancel the release.